content/posts/scientific-python/scientific-python-project/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Scientific Python: Community developed, community owned"
+date: 2022-08-14
+draft: false
+tags: ["Community", "Scientific-Python"]
+description: "Learn about the Scientific Python project."
+displayInList: true
+author: ["Juanita Gómez"]
+
+resources:
+  - name: featuredImage
+    src: "community.png"
+    params:
+      description: "Picture of the Scientific Python Community"
+      showOnTop: true
+---
+
+The Scientific Python project started as an initiative from Stéfan van der Walt and Jarrod Millman, to help coordinate the Scientific Python ecosystem and grow the community. Their idea is to improve communication between projects in the ecosystem, (get projects talking to one another) to plan strategies about their joint future, and to make that future a reality.
+
+## Why is this important?
+
+Initially, the Scientific Python developer community was small and it was easy to organize meetings to discuss important aspects of the ecosystem and take joint decisions. However, over the years the community has grown quickly and it has become harder, if not impossible, to have these meetings with all the members of the ecosystem and take decisions together. Each project now has their own community and there has been a lack of coordination that impacts negatively the users of the different tools. In the end, it is a fact that the different projects depend on one another, so it just makes sense to have some sort of coordination between them.
+
+## How are we doing this?
+
+### SPECs
+
+The first aim of the project, is to have a mechanism through which the community can establish cross-project policies. These policy documents are called [SPECs](https://scientific-python.org/specs/): Scientific Python Ecosystem Coordination documents. And they function similarly to PEPs, NEPs, SKIPs, or any of the other enhancement proposals—except that they will be relevant to multiple projects in the ecosystem.
+
+These documents will be recommendations written up by the community, and their authority will derive from endorsement by popular libraries. Some of them are already in progress and many are on the way!
+
+### Shared infrastructure
+
+The project also aims to provide a common engineering infrastructure to help maintainers. Some of the tools that are part of this are a [theme](https://github.com/scientific-python/scientific-python-hugo-theme) for the project websites, which is already been used by some of the projects (numpy and scipy), a self-hosted privacy-friendly web analytics platform, a shared [discussions forum](https://discuss.scientific-python.org), a [Scientific Python blog](https://blog.scientific-python.org), and project development statistics.
+
+### Developer events
```

Comment:
We could add 2 lines about summits.

Reply:
We definitely should. I will draft something tomorrow that @juanis2112 can edit to make sure the tone is consistent.

Reply:
I will also draft a sentence about the strategic plan, which is supposed to be the main focus of the grant that started the project. We still need to start working on that, but we've gotten distracted with getting infrastructure set up. However, now that the basic infrastructure is in place, we will start focusing more on the summits and the strategic plan. We will write up a separate post about the strategic plan later, but it would be good to at least mention it here.

Reply:
I do mention the summits in the developer events section but maybe it would be good to explain what they are. Is that what you mean @tupui and @jarrodmillman? If so, yeah feel free to add more about those things and I'm happy to review and add them.

Reply:
Maybe the first sentence just needs a rephrasing. Instead of events, it could just say summit. And we could put a link to the discuss section.

Reply:
Or the summit page: https://scientific-python.org/summits/

Reply:
I should expand that page a bit. For example, we are moving forward with the domain stack summit.